Update 22.vaccines.md (#909)
greenelab/covid19-review: content/22.vaccines.md

#### Novavax NVX-CoV2373

Novavax-CoV2373 is a protein nanoparticle vaccine candidate against SARS-CoV-2.
The vaccine is constructed from a mutated SARS-CoV-2 spike protein in combination with a specialized adjuvant to elicit an immune response against SARS-CoV-2.
The spike protein is recombinantly expressed in Sf9 insect cells[@doi: 10.1038/s41467-020-20653-8], which have previously been used for several other FDA-approved protein therapeutics [@doi:10.1021/acs.iecr.8b00985].

> The expressed spike protein contains mutations in the furin cleavage site (682-RRAR-685 to 682-QQAQ-685) to avoid cleavage of the spike protein as well as two proline substitutions (K986P and V987P) to improve thermostability [@doi: 10.1038/s41467-020-20653-8].

Reviewer: ```suggestion
The expressed spike protein contains mutations in the furin cleavage site (682-RRAR-685 to 682-QQAQ-685) to avoid cleavage of the spike protein as well as two proline substitutions (K986P and V987P) to improve thermostability [@doi:10.1038/s41467-020-20653-8].
```

The improved stability caused by the proline substitutions is particularly critical to facilitate global distribution, particularly to regions where local refrigerator/freezer capacities are limited.
Importantly, these amino acid substitutions did not affect the ability of the spike protein to bind the hACE2 receptor (the target receptor of SARS-CoV-2 spike protein).

> The Novavax-CoV2373 vaccine candidate uses a proprietary, saponin-based Matrix-MTM adjuvant that contains two different 40nm-sized particles formed by formulating purified saponin with cholesterol and phospholipids [@doi:10.1007/s12026-018-8991-x].

Reviewer 2: ```suggestion
The Novavax-CoV2373 vaccine candidate uses a proprietary, saponin-based Matrix-M^TM^ adjuvant that contains two different 40nm-sized particles formed by formulating purified saponin with cholesterol and phospholipids [@doi:10.1007/s12026-018-8991-x].
```

In preclinical models, the use Matrix-M adjuvant potentiated the cellular and humoral immune responses to influenza vaccines [@doi: 10.1016/j.vaccine.2013.01.039; @doi: 10.1016/j.vaccine.2009.09.044; @doi: 10.1111/j.1750- 2659.2011.00256.x; @doi: 10.1007/s12026-018-8991-x].

> Importantly, Matrix-M adjuvant-containing vaccines have shown acceptable safety profiles in human clinical trials [@doi: 10.1016/j.vaccine.2011.08. 042].

Reviewer: ```suggestion
Importantly, Matrix-M adjuvant-containing vaccines have shown acceptable safety profiles in human clinical trials [@doi:10.1016/j.vaccine.2011.08. 042].
```

In preclinical mouse models, Novavax-CoV2373 elicited high anti-spike IgG titers 21-28 days post-vaccination that can neutralize SARS-CoV-2 virus and protect the animals against virus challenge [@doi: 10.1038/s41467-020-20653-8]. 

> Antibody titers were significantly elevated in groups receiving the vaccine with the Matrix-M adjuvant compared to the groups without adjuvant. Novavax-CoV2373 was able to induce a multifunctional CD4/CD8 T-cell responses and generate high frequencies of follicular helper T-cells and B-cell germinal centers after vaccination.

Reviewer: ```suggestion
Antibody titers were significantly elevated in groups receiving the vaccine with the Matrix-M adjuvant compared to the groups without adjuvant.
Novavax-CoV2373 was able to induce a multifunctional CD4/CD8 T-cell responses and generate high frequencies of follicular helper T-cells and B-cell germinal centers after vaccination.
```

These findings were subsequently evaluated in a baboon primate model, in which Novavax-CoV2373 also elicited high antibody titers against the SARS-CoV-2 spike protein, as well as an antigen specific T-cell response.

> Based on this data Novavax initiated a Phase 1/2 clinical trial to evaluate the safety and immunogenicity of Novavax-CoV2373 with Matrix-M (NCT04368988)[@doi: 10.1056/NEJMoa2026920].

Reviewer: ```suggestion
Based on this data Novavax initiated a Phase 1/2 clinical trial to evaluate the safety and immunogenicity of Novavax-CoV2373 with Matrix-M (NCT04368988)[@doi:10.1056/NEJMoa2026920].
```
